packages/react-server/src/metrics/test/metric.test.tsx

```diff
@@ -1,46 +1,13 @@
-import {Server} from 'http';
+import {saddle, unsaddle} from 'saddle-up/koa';
 
-import React from 'react';
-import request from 'supertest';
-import getPort from 'get-port';
-
-import {createServer} from '../../server';
-import {mockMiddleware} from '../../test/utilities';
-
-jest.mock('@shopify/sewing-kit-koa', () => ({
-  middleware: jest.fn(() => mockMiddleware),
-  getAssets() {
-    return {
-      styles: () => Promise.resolve([]),
-      scripts: () => Promise.resolve([]),
-    };
-  },
-}));
+import {metricsMiddleware} from '../metrics';
 
 describe('createServer()', () => {
-  let server: Server;
-  let port: number;
-  const ip = 'http://localhost';
-
-  beforeEach(async () => {
-    port = await getPort();
-
-    server = await createServer({
-      port,
-      ip,
-      render: () => <div>Mock app</div>,
-    });
-  });
-
-  afterEach(() => {
-    server.close();
-  });
+  afterAll(unsaddle);
 
   it('responds with a Server-Timing header', async () => {
-    const response = await request(`${ip}:${port}`)
-      .get('/')
-      .then((resp: request.Response) => resp);
-
-    expect(response.header['server-timing']).toBeDefined();
+    const wrapper = await saddle(metricsMiddleware);
```

Comment:
Is `wrapper` the right word to use in this context? Would `server` be better?

Reply:
it isn't a server, it's a `SaddleUp` instance. I guess `server` is fine